ocaml/src/asd_server.ml

```diff
@@ -1646,7 +1653,7 @@ let run_server
                  ~tcp_keepalive
                  ~tls:None protocol
        in
-       t :: threads
+       (wrap_t t "plain server") :: threads
```

Comment:
This change wasn't needed, I added it while debugging some problem, but in the end it turned out the asds weren't to blame.
Still thought the change could be useful so I left it in.